bundles/org.openhab.core.config.discovery.addon/src/test/java/org/openhab/core/config/discovery/addon/tests/AddonSuggestionServiceTests.java

```diff
@@ -125,13 +126,13 @@ private void setupMockAddonInfoProvider() {
         AddonDiscoveryMethod hp = new AddonDiscoveryMethod().setServiceType(AddonFinderConstants.SERVICE_TYPE_MDNS)
                 .setMatchProperties(
                         List.of(new AddonMatchProperty("rp", ".*"), new AddonMatchProperty("ty", "hp (.*)")))
-                .setMdnsServiceType("_printer._tcp.local.");
+                .setParameters(List.of(new AddonParameter("mdnsServiceType", "_printer._tcp.local.")));
```

Comment:
ditto

Reply:
Done, but with local constant. Otherwise an import of MDNSAddonFinder is required. This would create a circular import.